src/data/nat/totient.lean

```diff
@@ -335,4 +335,42 @@ begin
   rw [sub_mul, one_mul, mul_comm, mul_inv_cancel hp', cast_pred hp],
 end
 
+@[to_additive sum_factors_gcd_add_sum_factors_mul]
```

Comment:
Can you move this to some other file? I don't know where exactly, but it seems a little weird it is in `totient.lean`.